src/disk_manager.rs

```diff
@@ -246,9 +246,9 @@ fn listen(
 
 fn respond_to_client<T: protobuf::Message>(result: &T, s: &mut Socket) -> BynarResult<()> {
     let encoded = result.write_to_bytes()?;
-    let msg = Message::from_slice(&encoded)?;
-    debug!("Responding to client with msg len: {}", msg.len());
-    s.send_msg(msg, 0)?;
+    //let msg = Message::from_slice(&encoded)?;
```

Comment:
If these commented lines aren't needed anymore than they should be deleted.

Reply:
Fixed!